src/Bicep.Core/Syntax/SyntaxTree.cs

```diff
@@ -5,10 +5,11 @@
 using System.Collections.Immutable;
 using Bicep.Core.Parsing;
 using Bicep.Core.Text;
+using Bicep.Core.Workspaces;
 
 namespace Bicep.Core.Syntax
 {
-    public class SyntaxTree
+    public class SyntaxTree : ISourceFile
```

Comment:
Per our discussion offline, I'm going to rename the class to `BicepFile` in a follow-up PR.